src/main/java/org/folio/oaipmh/mappers/AbstractMapper.java

```diff
@@ -7,24 +14,18 @@
 import org.marc4j.marc.Record;
 import org.w3c.dom.Node;
 
-import javax.xml.transform.dom.DOMResult;
-import java.io.*;
-import java.nio.charset.StandardCharsets;
 
-/**
- * Converts MarkJson format to MarcXML format.
- */
-public class MarcXmlMapper implements Mapper {
+public abstract class AbstractMapper implements Mapper {
```

Comment:
Your abstract class doesn't have any abstract methods...

Reply:
I think that if you're going to have an AbstractMapper class it should be generic; not specific to MARC.  

Consider the where we might need to create a mapper for something NOT derived from MarcXML... say the FOLIO Inventory json records for instance...  

If you think there is some shared/common code, I would keep it in this Base/abstract mapper class and extract the MARC-specific code out into the MarcXmlMapper class.  

If there isn't any shared/common code, then there's no need for the AbstractMapper.  Just have each mapper implement the Mapper interface. 

For the DC mapper, which builds upon the MarcXmlMapper, that's fine, just have it extend that class